gravitee-apim-console-webui/src/management/instances/details/monitoring/instance-monitoring.html

```diff
@@ -51,7 +51,7 @@ <h1><ng-md-icon size="48" icon="error"></ng-md-icon> There is no data for stoppe
       </md-card-content>
     </md-card>
   </div>
-  <md-card>
+  <md-card data-testid="jvm-box">
```

Comment:
Like as discuss this morning, we could use a more explicit data-testid value something like of `instance-monitoring_jvm-box`.

And perhap's improve the README.md with some wording rules.